src/server/src/project-templates/services/project-templates.service.ts

```diff
@@ -19,6 +19,7 @@ export class ProjectTemplatesService extends TypeOrmCrudService<ProjectTemplate>
       .innerJoinAndSelect("projectTemplate.projects", "projects")
       .innerJoinAndSelect("projects.user", "user")
       .where("organization.slug = :slug", { slug: slug })
+      .where("projectTemplate.isActive = :true", { true: true })
```

Comment:
From the documentation for `.where(..)`:
> If you had previously WHERE expression defined, calling this function will override previously set WHERE conditions.

You can get around this by using `.andWhere(...)` instead, or combining this into the above `.where(...)`. Also an additional cleanup - no need to pass in `true` as a parameter - just use the sql keyword `TRUE`.

```suggestion
      .where("organization.slug = :slug", { slug: slug })
      .andWhere("projectTemplate.isActive = TRUE")
```

or maybe 

```suggestion
      .where("organization.slug = :slug AND projectTemplate.isActive = TRUE", { slug: slug })
```